README.md

```diff
@@ -43,16 +43,34 @@ go build -o otel
 spin pluginify --install
 ```
 
+# Observability Stacks
+
+The `otel` plugin currently supports two different observability stacks:
+
+- Default: Multi-container observability stack based on Prometheus, Loki, Grafana and Jaeger
+- Aspire: Single-container observability stack using .NET Aspire Dashboard
+  
 # Usage
 
 Once the plugin is installed, you can try the below commands:
 
-## Set up the dashboards
+## Set up the observability stack
+
+You chose the desired observability stack as part of the `setup` command. You can change the observability stack at any point in time by cleaning it up and re-running the `setup` command using another stack.
```

Comment:
Nit: Maybe we should demo running `cleanup`?

Reply:
@calebschoepp there is a section at the end of the README showing how to clean up:

https://github.com/fermyon/otel-plugin?tab=readme-ov-file#terminate-the-dashboards